.github/workflows/build.yml

```diff
@@ -15,15 +15,17 @@ jobs:
   create-runner:
     uses: ./.github/workflows/create_runner.yml
     secrets: inherit
+    concurrency:
+      group: ${{ github.ref == 'refs/heads/main' && github.run_id || github.sha }}-create-runner
+      cancel-in-progress: false
 
   configuration:
     name: Configure job parameters
     runs-on: ${{ needs.create-runner.outputs.label }}
     needs: [create-runner]
     concurrency:
-      # github.head_ref is set by a pull_request event - contains the name of the source branch of the PR
-      # github.ref_name is set if the event is NOT a pull_request - it contains only the branch name.
-      group: ${{ github.head_ref || github.ref_name }}-configuration
+      group: ${{ github.ref == 'refs/heads/main' && github.run_id || github.sha }}-configuration
+      cancel-in-progress: true
```

Comment:
Why is it ok to cancel here? E.g., are we skipping the build step, then, later? How does it work? :thinking:

Reply:
This only applies if you force-push the same commit.

Reply:
:bulb:

Reply:
I don't think it would cancel on force push, because the SHA is different then.